app/helpers/interuption_screen_helper.rb

```diff
@@ -0,0 +1,16 @@
+module InteruptionScreenHelper
+  def display_informative_text(informative_text, case_log)
+    translation_questions = informative_text["argument"].map { |x| case_log.form.get_question(x) }
+    begin
+      case translation_questions.count
```

Comment:
Can this never be more than 2?

Reply:
It potentially could be but realistically probably won't be - I suggested that we just go with the YAGNI principle here and don't extend it beyond 2 until we actually end up needing to. Happy to change this if you think we should

Reply:
Would the behaviour be different for 2 vs 3?

i.e. rather than a case statement just a `translation_questions.count > 1`?